core/src/helpers/mod.rs

```diff
@@ -173,22 +178,24 @@ pub fn replace_env_variable_to_raw_name(rpc: &str) -> String {
     }
 }
 
-pub fn u256_to_i256(value: U256) -> I256 {
-    let max_i256_as_u256 = U256::from_str_radix(
-        "7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
-        16,
-    )
-    .unwrap();
-
-    if value <= max_i256_as_u256 {
-        // If the value is less than or equal to I256::MAX, it's a positive number
-        I256::from_raw(value)
-    } else {
-        // If it's larger, it represents a negative number in two's complement
-        let twos_complement = (!value).overflowing_add(U256::one()).0;
-        I256::from_raw(twos_complement).wrapping_neg()
-    }
-}
+// Remove this if we're sure we no longer need it after the Alloy migration
+//
+// pub fn u256_to_i256(value: U256) -> I256 {
```

Comment:
this was due to uniswap contracts of twos_complement is this now fixed aka `-` numbers in db?